Formula/doxygen.rb

```diff
@@ -23,7 +23,7 @@ class Doxygen < Formula
   depends_on "cmake" => :build
   depends_on "graphviz" => :optional
   depends_on "qt5" => :optional
-  depends_on "llvm" => "with-clang" if build.with? "libclang"
+  depends_on "llvm" if build.with? "libclang"
```

Comment:
1. So, do you want these changes in this PR or you'll add them separately?
2. Do you mean something like:
```ruby
deprecated_option "with-libclang" => "with-llvm"
```
?